scala/spark-wordcount/README.md

```diff
@@ -2,27 +2,41 @@
 
```

Comment:
Please update title to specify that this uses the RDD api

Reply:
fixed